File: C_Programs/CPC/Lab 4/P8.c
#include <stdio.h>

int main(){
    int n = 0, i = 0, sum = 0;
    printf("Enter the amount of numbers: ");
    scanf("%d", &n);
    int num[n];
    for(i = 0; i < n; i++){
        printf("Enter number %d: ", i + 1);
        scanf("%d", &num[i]);
    }
    for(i = 0; i < n; i++){
        int temp = num[i];
        sum += (temp * (i + 1));
    }
    printf("Weighted Sum: %d\n", sum);
}

File: Python_Programs/School_practicals/P8.py
x = int(input("Enter a number: "))
n = int(input("Enter the final exponent for the series: "))
sum = count = 1
while count <= n:
    if (count % 2) == 0:
        sum += x**count
    else:
        sum -= x**count
    count += 1
print("Sum: ", sum)